src/v2/controllers/FormController.js

```diff
@@ -147,6 +149,17 @@ export default Controller.extend({
 
     this.toggleFormButtonState(true);
     model.trigger('request');
+
+    // NOTE: okta-idx-js does not know whether a remediation
+    // is required to redirect or perform a GET request.
+    // As a result, we handle this accordingly by checking the formName
+    const fullPageRedirectForms = [ FORMS.REDIRECT_IDP ];
```

Comment:
Since this variable is intended for only the `handleSaveForm` method, I'd prefer to keep this as function scope instead of global. That ok?

Reply:
@jmelberg-okta did you explore the possibility of triggering invokeAction instead of using default save ? We should put this code under `handleInvokeAction`.. We don't need to run this code for every form submit that doesn't make any sense.. This is special case so handle this as a special case in `handleInvokeAction` and should be triggered only when it's required..